concent_api/core/validation.py

```diff
@@ -58,7 +59,23 @@ def validate_id_value(value, field_name):
         )
 
 
-def validate_public_key(value, field_name):
+def validate_hex_public_key(value, field_name):
```

Comment:
it is dduplication of the function below. Please make one generic function with two additional parameters:
`type` and `desired_key_length`